src/utils/user.test.js

```diff
@@ -8,17 +8,31 @@ import {
 } from './user'
 
 const {
-  AGENTS_GOOGLE_GROUPNAME,
   CONTRACT_MANAGERS_GOOGLE_GROUPNAME,
   AUTHORISATION_MANAGERS_GOOGLE_GROUPNAME,
   OPERATIVES_GOOGLE_GROUPNAME,
-  CONTRACTORS_GOOGLE_GROUPNAME,
   BUDGET_CODE_OFFICER_GOOGLE_GROUPNAME,
+  DLO_CONTRACTOR_GOOGLE_GROUPNAME,
 } = process.env
 
+const CONTRACTOR_GROUP_NAME = 'repairs-hub-contractors-alphatrack-staging'
+const AGENT_GROUP_NAME = 'repairs-hub-agents-staging'
+
 describe('buildUser', () => {
```

Comment:
The previous tests for Agents and Contractors were checking for specific group names, rather than regex. I have replaced these with correct tests